tests/test_utils.py

```diff
@@ -23,36 +25,56 @@ def test_load():
 
 def test_close_matches():
     # test underspecified
-    assert set(randomname.util.close_matches('music')) == {
-        'verbs/music',
-        'nouns/music_theory',
-        'adjectives/music_theory',
-        'nouns/music_production',
-        'verbs/music_production',
-        'nouns/music_instruments',
-    }
+    matches = randomname.util.close_matches('music')
+    assert {Path(m) for m in matches} == {
```

Comment:
as `close_matches` return file paths we also need to compare file paths which depending on the OS might look different.

Reply:
Instead of `close_matches` returning paths, maybe instead we should convert `AVAILABLE` using `.replace(os.sep, '/')`

Reply:
and `ALL_CATEGORIES`. Idk that way it corresponds with what the user enters. And then `as_valid_path` makes sure everything gets coerced to a path before loading.

Reply:
So I'd recommend doing this:
```python
def getallcategories(d=''):
    d = os.path.join(WORD_PATH, d)
    return [
        os.path.relpath(os.path.splitext(f)[0], d).replace(os.sep, '/') # ++ .replace(os.sep, '/')
        for f in glob.glob(os.path.join(d, '**/*.txt'), recursive=True)]

```

Reply:
done 👍🏻